docs/src/kubernetes/zeebe-operator.md

```diff
@@ -0,0 +1,91 @@
+
+> kubectl get zb
+
+If you delete the ZeebeCluster resource the actual ZeebeCluster will be automatically removed from your cluster. 
+Now you can check that there is a new “Namespace” created with:
+
+> kubectl get ns
+
+And also check that the cluster is correctly provisioned by looking at the Pods created inside the newly created namespace with
+
+> kubectl get pods -n <Zeebe Cluster Name> -w
+
+The next video show these commands in action along with the installation of the Zeebe Kubernetes Operator:
+
+![Intro video](https://www.youtube.com/watch?v=U-crhMfuJgY)
+
+
+## Technical Details and Dependencies
+
+This Kubernetes Operator was built using KubeBuilder V2.1+, Tekton 0.8.0+ and Helm 3.
+
+The Operator Defines currently 1 CRD (Custom Resource Definition): `ZeebeCluster`, but in future versions, new types will be defined for other components such as Zeebe Operate and Workers.  The ZeebeCluster resource represent a low-level resource which will instantiate a Zeebe Cluster based on predefined parameters. This low-level resource definition can be used to define the cluster topology and HA configurations.
+
+The Zeebe Kubernetes Operator was built using the [kubebuilder framework](https://github.com/kubernetes-sigs/kubebuilder) for writing the controller’s logic and scaffolding the CRD type. Internally it does interact with [Tekton Pipelines](https://github.com/tektoncd/pipeline) in order to install and manage Zeebe Helm Charts.  The project itself is being built, released and tested using [Jenkins X](https://jenkins-x.io/). This lead to some changes in how KubeBuilder’s project is structured, as in it current shape the project is not organised in a way that is easy to create a Helm Chart out of it.
+
+The main flow of the Operator works like this: 
+![Flow](/kubernetes/zeebe-operator-flow.png)
+
+
+First, the Operator will be constantly looking for ZeebeCluster resources, when one is found a new Namespace is created and a Tekton Task and TaskRun are created to “upgrade” the Helm Charts defined inside the Version Stream repository (hosted here: https://github.com/zeebe-io/zeebe-version-stream-helm ).
+
+This repository (referred as Version Stream Repository) contains a list of blessed versions that will be installed when a new ZeebeCluster resource is detected by the operator. Using a Version Stream Repository provide us with the flexibility to evolve the operator code and the charts that define what needs to be provisioned independently. Allowing a simple upgrade path to future versions by using a Git repository as central reference to a stable version.
+
+In future versions, the option to choose a version stream repository will be provided, allowing different streams.
+
+The Task created in Tekton Pipelines execute two basic operations:
+
+- First, clone Version Stream Repository (using simple git clone)
+Run Helm Upgrade of the chart defined in the Version Stream Repository (it does actually an upgrade/install if it doesn’t exist)
```

Comment:
```suggestion
Run Helm Upgrade of the chart defined in the Version Stream Repository (it will automatically upgrade/install if it doesn’t exist)
```